src/common/dbconfig.cpp

```diff
@@ -0,0 +1,31 @@
+#include "common/dbconfig.h"
+
+#include <fstream>
+#include <iostream>
+
+namespace biscuit::common {
+
+json DBConfig::ReadConfig(const char *path) {
+  std::ifstream fin = std::ifstream(path);
+  if (fin.good()) {
+    json j;
+    fin >> j;
+    return j;
+  }
+  return nullptr;
+}
+
+json DBConfig::config = nullptr;
+bool DBConfig::cached = false;
+
+const json &DBConfig::GetConfig() {
+  if (!cached) {
+    config = ReadConfig(CONFIG_PATH);
+    cached = true;
+  }
```

Comment:
We should explicitly handle the `nullptr` else it will cause a runtime error.
I am thinking about calling `ReadConfig` from main but considering the whole `static` mess, I guess it's not feasible. Let me know what you think.

Reply:
[update] line 8-25

Reply:
lets throw an Exception...
This will let the user know during bootstrapping.

Reply:
Cool